frontend/src/components/inputs/CustomTextField.js

```diff
@@ -0,0 +1,17 @@
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
```

Comment:
Nit: Do we need to care about the positioning of the imports alphabetically?

Reply:
For the pleasure of the eyes, yes, will rearrange it haha